doc/classes/SpatialMaterial.xml

```diff
@@ -299,11 +299,14 @@
 			The number of vertical frames in the particle sprite sheet. Only enabled when using [constant BILLBOARD_PARTICLES]. See [member params_billboard_mode].
 		</member>
 		<member name="proximity_fade_distance" type="float" setter="set_proximity_fade_distance" getter="get_proximity_fade_distance">
-			Distance over which the fade effect takes place. The larger the distance the longer it takes for an object to fade.
+			The distance over which the fade effect takes place. The larger the distance, the longer it takes for an object to fade. See also [member proximity_fade_far_distance].
 		</member>
 		<member name="proximity_fade_enable" type="bool" setter="set_proximity_fade" getter="is_proximity_fade_enabled" default="false">
 			If [code]true[/code], the proximity fade effect is enabled. The proximity fade effect fades out each pixel based on its distance to another object.
 		</member>
+		<member name="proximity_fade_far_distance" type="float" setter="set_proximity_fade_far_distance" getter="get_proximity_fade_far_distance">
+			If set to a value greater than [code]0.0[/code], this will fade the material as it gets further away from other solid materials. This can be used things such as decals, "cutting torch" effects and shockwave/explosion effects. The larger the distance, the longer it takes for an object to fade. See also [member proximity_fade_distance].
```

Comment:
```suggestion
			If set to a value greater than [code]0.0[/code], this will fade the material as it gets further away from other solid materials. This can be used for things such as decals, "cutting torch" effects and shockwave/explosion effects. The larger the distance, the longer it takes for an object to fade. See also [member proximity_fade_distance].
```